pkg/executables/clusterctl.go

```diff
@@ -172,12 +172,16 @@ func (c *Clusterctl) BackupManagement(ctx context.Context, cluster *types.Cluste
 	return nil
 }
 
-func (c *Clusterctl) MoveManagement(ctx context.Context, from, to *types.Cluster) error {
+func (c *Clusterctl) MoveManagement(ctx context.Context, from, to *types.Cluster, clusterName string) error {
 	params := []string{"move", "--to-kubeconfig", to.KubeconfigFile, "--namespace", constants.EksaSystemNamespace}
 	if from.KubeconfigFile != "" {
 		params = append(params, "--kubeconfig", from.KubeconfigFile)
 	}
 
+	if clusterName != "" {
```

Comment:
do we have a usecase to move all clusters?
I would lean to make this argument mandatory. That would avoid errors where we end up passing an empty cluster name my accident.

And if we ever need to move all clusters, we just create a `MoveAllClusters` method. Which is more explicit.

Reply:
Sure I like that